components/service/glean/src/main/java/mozilla/components/service/glean/config/Configuration.kt

```diff
@@ -36,7 +37,7 @@ data class Configuration internal constructor(
     // NOTE: since only simple object or strings can be made `const val`s, if the
     // default values for the lines below are ever changed, they are required
     // to change in the public constructor below.
-    val httpClient: Lazy<Client> = lazy { HttpURLConnectionClient() },
+    val httpClient: Lazy<PingUploader> = lazy { ConceptFetchHttpUploader(lazy { HttpURLConnectionClient() }) },
```

Comment:
Do we really need the nested lazyness here?

Reply:
Good point. I'm making this a non-lazy `PingUploader`, leaving the lazy part as an implementation detail for concept-fetch.